packages/flet_local_auth/lib/src/local_auth.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:io' show Platform;
+
+import 'package:flet/flet.dart';
+import 'package:flutter/widgets.dart';
+import 'package:local_auth/local_auth.dart';
+
+class LocalAuthenticationControl extends StatefulWidget {
+  final Control? parent;
+  final Control control;
+  final FletControlBackend backend;
+
+  const LocalAuthenticationControl(
+      {super.key,
+      required this.parent,
+      required this.control,
+      required this.backend});
+
+  @override
+  State<LocalAuthenticationControl> createState() =>
+      _LocalAuthenticationControlState();
+}
+
+class _LocalAuthenticationControlState
+    extends State<LocalAuthenticationControl> {
+  @override
+  Widget build(BuildContext context) {
+    debugPrint("LocalAuthentication build: ${widget.control.id}");
+
+    () async {
+      final LocalAuthentication auth = LocalAuthentication();
+
+      widget.backend.subscribeMethods(widget.control.id,
+          (methodName, args) async {
+        switch (methodName) {
+          case "available":
+            // get's available biometrics
+            final List<BiometricType> availableBiometrics =
+                await auth.getAvailableBiometrics();
```

Comment:
Can we make it possible (by creating a new case) to do something like `auth.get_available_biometrics()` from python, and get as result these BiometricTypes?
I find it smh useful.

Reply:
If you cant, let me know. 
I can help with that.

Reply:
let me just check it out.

I will ask your help if I couldn't make it.

Reply:
Ok, we can leave this out then.